tests/all/integration/api/helpers/test_scheduled_jobs.py

```diff
@@ -29,3 +30,17 @@ def test_event_invoices_mark_due(self):
 
             self.assertEqual(status_new, "due")
             self.assertNotEqual(status_paid, "due")
+
+    def test_delete_ticket_holders_with_no_order_id(self):
+        """Method to test deleting ticket holders with no order id after expiry time"""
+        with app.test_request_context():
+            ticket_holder_new = TicketHolder(firstname='John', lastname='Doe')
+
+            db.session.commit()
+
+            ticket_holder_id = ticket_holder_new.id
+
+            delete_ticket_holders_no_order_id()
```

Comment:
Also for here, how do I add the delay of order-expiry time for deleting this entity?

Reply:
You need to pass past created_at